reflex/components/graphing/plotly.py

```diff
@@ -25,7 +25,7 @@ class Plotly(PlotlyLib):
     is_default = True
 
     # The figure to display. This can be a plotly figure or a plotly data json.
-    data: Var[Figure]
+    data: Var[Any]
```

Comment:
```suggestion
    data: Var["Figure"]
```

And then at the top of the file, do something like

```python
try:
    from plotly.graph_objects import Figure
except ImportError:
    Figure = Any
```

That way the type checking at least works nicer when you do have plotly installed.